gu-hub-api.yaml

```
@@ -0,0 +1,582 @@
+        404:
+          description: 'Session not found'
+
+  /sessions/{sessionId}/blob/{blobId}:
+    parameters:
+      - $ref: '#/parameters/sessionId'
+      - name: blobId
+        description: 'Blob identifier'
+        type: string
+        in: path
+        required: true
+    put:
+      tags:
+        - lob-man
+      operationId: uploadBlob
+      parameters:
+        - name: body
+          in: body
+          schema:
+            type: string
+            format: binary
+      consumes:
+        - application/octet-stream
+      responses:
+        200:
+          description: OK
+    get:
+      tags:
+        - lob-man
+      operationId: downloadBlob
+      responses:
+        200:
+          description: OK
+          schema:
+            type: file
+
+
+definitions:
+  PeerInfo:
+    description: General information about GU subnetwork node
+    required:
+      - peerAddr
+    properties:
+      nodeName:
+        type: string
+        description: Node name or hostname from reverse dns if node name is not set
+      peerAddr:
+        type: string
+        description: 'node address from current connection formated as <ip-addr>:<port>'
+      nodeId:
+        type: string
+        description: node public key hash in etherium format
+        example: '0x875f272d3b9e7b55a5784a131a60bf3d7a42c73c'
+        readOnly: true
+      tags:
+        type: array
+        items:
+          type: string
+  HubSession:
+    properties:
+      id:
+        type: string
+        description: Uniqe HUB Session identifier
+        example: 47f1dc88-0282-4ac3-f765-02b303e8c278
+        readOnly: true
+      created:
+        type: string
+        description: Session creation timestamp
+        format: date-time
+        readOnly: true
+      expires:
+        type: string
+        format: date-time
+      allocation:
+        type: string
+        description: resource allocation mode
+        enum:
+          - manual
+          - auto
+      name:
+        type: string
+        description: optional human readable name
+      tags:
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+          pattern: '^[a-zA-Z][a-zA-Z0-9_:-]*$'
+  PeerSessionSpec:
+    properties:
+      envType:
+        $ref: '#/definitions/EnvType'
+      image:
+        properties:
+          hash:
+            type: string
+            description: Consistency hash
+          url:
+            type: string
+            description: Image location spec
+      name:
+        type: string
+        description: human readable name
+      tags:
+        type: array
+        description: peer session tags
+        uniqueItems: true
+        items:
+          type: string
+  Command:
+    type: object
+    properties:
+      exec:
+        $ref: '#/definitions/ExecCommand'
+      start:
+        $ref: '#/definitions/StartCommand'
+      stop:
+        $ref: '#/definitions/StopCommand'
+      addTags:
+        type: array
```

I'd introduce `TagsCommand` and use it here and within `delTags` below
```suggestion
        $ref: '#/definitions/TagsCommand'
```